README.md

```diff
@@ -4,6 +4,14 @@ This repository is associated with the [Distributed Trace Context Community Grou
 
 Specification for tracing context propagation format.
 
+## Team Communication 
+
+Overview of team related communication channels
+
+* GitHub issues for any specification related issues
+* Mailing List for general discussions. Please subscribe to [public-trace-context@w3.org](public-trace-context@w3.org)
```

Comment:
recommend this as the link instead, where people can actually subscribe: http://lists.w3.org/Archives/Public/public-trace-context/